src/main/fc/core.c

```diff
@@ -325,6 +325,12 @@ void updateArmingStatus(void)
             unsetArmingDisabled(ARMING_DISABLED_ANGLE);
         }
 
+        if (flipOverAfterCrashActive && !IS_RC_MODE_ACTIVE(BOXFLIPOVERAFTERCRASH)) {
+            setArmingDisabled(ARMING_DISABLED_CRASHFLIP);
+        } else {
+            unsetArmingDisabled(ARMING_DISABLED_CRASHFLIP);
+        }
```

Comment:
This will never be true (see line 290), so only `unsetArmingDisabled(...)` in the else-block can ever be triggered.

Reply:
Yes.  OK.  Thanks.  :-)
I don't know why crashflip is handled differently, in core.c, from other box 'modes', but it sure is confusing!  It gets forced to 'off' when disarmed, but previously was not turned off if the switch was turned off while armed, only a disarm could turn it off.  I suppose that was intentional.
I think it is acceptable to turn it off while armed so long as throttle is zero.  That's the normal 'next step' for the user.
Perhaps in a later PR you could clean it up?
Anyway, the behaviour is the same without those lines, so I removed them.  Thanks.